cmd/talosctl/cmd/talos/interactive-installer.go

```diff
@@ -0,0 +1,68 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package talos
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/docker/distribution/context"
+	"github.com/spf13/cobra"
+
+	"github.com/talos-systems/talos/pkg/machinery/api/machine"
+	"github.com/talos-systems/talos/pkg/machinery/client"
+)
+
+var interactiveInstallerCmdFlags struct {
+	filename string
+}
+
+// interactiveInstallerCmd represents a command.
+var interactiveInstallerCmd = &cobra.Command{
+	Use:   "interactive-installer",
+	Short: "Interact with the interactive installer",
+	Long:  ``,
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if interactiveInstallerCmdFlags.filename == "" {
+			return fmt.Errorf("no filename supplied for configuration")
+		}
+
+		ctx := context.Background()
+
+		addr := args[0]
+
+		c, err := client.NewInsecureTokenClient(ctx, addr)
```

Comment:
It can be cool if it's possible to add a new method called  `WithClientInsecure`, or introduce `insecure` flag into `WithClient` method, to get a command for configuration apply.

I think it can be cool if apply config just works both for insecure mode and secure mode.

```
talosctl apply-config --insecure init.yaml
```
Even better approach would be to avoid exposing this  `--insecure` flag, but detect if node runs in insecure mode and select it automatically.

And for interactive mode it can be:
```
talosctl apply-config --insecure --interactive
```

Reply:
Yeah I was thinking about that but then I thought that this subcommand might be slightly different because it could do more than just apply the config. It could, for example, be an interactive exchange between the user that prompts the user for input parameters, and then even generates the config server side and hands them off to the user. If that isn't clear, I can clarify in a meet.